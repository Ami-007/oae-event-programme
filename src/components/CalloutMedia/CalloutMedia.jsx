import "./CalloutMedia.scss";
import React from 'react'

const CalloutMedia = () => {
  return (
    <div>
      <h1>works</h1>
    </div>
  )
}

export default CalloutMedia;
