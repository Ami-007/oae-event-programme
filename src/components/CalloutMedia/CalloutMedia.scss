@use "../../assets/sass/variables" as *;
@use "../../assets/sass/typography" as *;

.callout-media {
  background: linear-gradient(
    180deg,
    #ffff 25%,
    $modern-primary 25%,
    $modern-primary 25%,
    $modern-primary 25%
  );
  .image-video-container {
    background: none;
  }
  .orchestra-quote {
    background: $modern-primary;
    padding-top: 2.813rem;
    padding-bottom: 5rem;
  }
}
