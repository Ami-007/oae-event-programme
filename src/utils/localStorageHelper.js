export const getDarkTheme = () => {
  //   if (darkTheme == true) {
  //     localStorage.setItem("--background-color", darkTheme);
  //   } else {
  //     localStorage.setItem("theme", "light");
  //   }
  console.log(localStorage.getItem("darkMode"));

  // convert isDarkMode to a Boolean
  // return that boolean

  return true;
};

export const setDarkTheme = (isDarkMode) => {
  localStorage.setItem("darkMode", isDarkMode.toString());
};
