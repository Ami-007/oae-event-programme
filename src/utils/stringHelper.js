export const getFontType = (fontType) => {
  if (typeof fontType === "string") {
    fontType = parseInt(fontType);
  }
  if (fontType === 1) {
    return "modern";
  } else if (fontType === 2) {
    return "classic";
  } else {
    return "modern";
  }
};

export default getFontType;
