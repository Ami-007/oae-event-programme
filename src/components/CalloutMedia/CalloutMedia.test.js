import CalloutMedia from "./CalloutMedia";
import { customRender } from "../../utils/testUtils";

test("Renders Callout Media component wih default page", () => {
  const { container } = customRender(
    <CalloutMedia content="123">
      <p>sample children</p>
    </CalloutMedia>
  );
  expect(container).toMatchSnapshot();
});
