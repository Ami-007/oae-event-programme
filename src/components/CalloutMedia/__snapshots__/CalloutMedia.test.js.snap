// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Callout Media component wih default page 1`] = `
<div>
  <div
    class="callout-media"
  >
    <div
      class="image-video-container"
    >
      <img
        alt="Event image"
        class="header-image"
        src="123"
      />
    </div>
    <div>
      <p>
        sample children
      </p>
    </div>
  </div>
</div>
`;
